content/200-orm/200-prisma-client/500-deployment/301-edge/485-deploy-to-vercel.mdx

```diff
@@ -0,0 +1,458 @@
+---
+title: 'Deploy to Vercel Edge Functions & Middleware'
+navTitle: 'Deploy to Vercel'
+metaTitle: 'Deploy to Vercel Edge Functions'
+metaDescription: 'Learn the things you need to know in order to deploy an Edge function that uses Prisma Client for talking to a database.'
+tocDepth: 3
+preview: true
+---
+
+<TopBlock>
+
+This page covers everything you need to know in order to deploy an edge function to [Vercel Edge Functions](https://vercel.com/docs/functions/edge-functions) that uses Prisma Client for talking to a database.
+
+</TopBlock>
+
+## General considerations when deploying to Vercel Edge Functions
+
+### Using an edge-compatible driver
+
+When deploying a Vercel Edge Function that uses Prisma ORM, you need to use an [edge-compatible driver](/orm/prisma-client/deployment/edge/overview#edge-compatibility-of-database-drivers) and its respective [driver adapter](/orm/overview/databases/database-drivers#driver-adapters) for Prisma ORM.
```

Comment:
This could be rewritten in a way that makes clear that this is not on Prisma, but on Vercel to support these drivers. We can only support the ones that work on Vercel, and those are the three that are listed below.

Reply:
Rewrote the section to make this more clear:

---


Vercel's Edge Runtime currently only supports a limited set of database drivers:

- [Neon Serverless](https://neon.tech/docs/serverless/serverless-driver) uses HTTP to access the database (also compatible with [Vercel Postgres](https://vercel.com/docs/storage/vercel-postgres))
- [PlanetScale Serverless](https://planetscale.com/docs/tutorials/planetscale-serverless-driver) uses HTTP to access the database
- [`@libsql/client`](https://github.com/tursodatabase/libsql-client-ts) is used to accessed Turso databases

Note that [`node-postgres`](https://node-postgres.com/) (`pg`) is currently _not_ supported on Vercel Edge Functions.

When deploying a Vercel Edge Function that uses Prisma ORM, you need to use one of these [edge-compatible drivers](/orm/prisma-client/deployment/edge/overview#edge-compatibility-of-database-drivers) and its respective [driver adapter](/orm/overview/databases/database-drivers#driver-adapters) for Prisma ORM.